osu.Game/Online/API/Requests/SearchBeatmapSetsRequest.cs

```diff
@@ -11,17 +12,28 @@ namespace osu.Game.Online.API.Requests
 {
     public class SearchBeatmapSetsRequest : APIRequest<SearchBeatmapSetsResponse>
     {
+        public class Cursor
+        {
+            [JsonProperty("_id")]
+            public string Id;
+
+            [JsonProperty("approved_date")]
+            public string ApprovedDate;
+        }
```

Comment:
As we were discussing earlier this won't work if sort criteria are changed - web will return different fields.